compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1376,6 +1376,8 @@ pub struct TargetOptions {
     pub is_like_msvc: bool,
     /// Whether a target toolchain is like WASM.
     pub is_like_wasm: bool,
+    /// Whether a target toolchain is like Android.
```

Comment:
It'd probably be good to expand this comment a little more. What does being like Android entail? How does that differ from a plain Linux target, for example?

Reply:
Good idea.  I've added some additional information.  Let me know if you think I should add more.

Reply:
Looks good to me!